modules/microshift-data-cleaning-certs.adoc

```diff
@@ -42,9 +42,4 @@ Cleanup succeeded
 The {microshift-short} service is stopped after you run the script.
 ====
 
-. Restart the {microshift-short} service by running the following command:
-+
-[source,terminal]
-----
-$ sudo systemctl start microshift
-----
\ No newline at end of file
+. Run the `dnf delete microshift` command.
```

Comment:
We should not make changes to this module as it is used elsewhere. I think we need a new one that is just for uninstalling microshift.